packages/arb-token-bridge-ui/src/components/common/WalletConnectChainDropdown.tsx

```diff
@@ -0,0 +1,66 @@
+import { Chain } from 'wagmi'
+import { useMemo } from 'react'
+
+import { TargetChainKey } from '../../util/WalletConnectUtils'
+import { getSupportedNetworks } from '../../util/networks'
+import { getWagmiChain } from '../../util/wagmi/getWagmiChain'
+import { NetworkListbox } from '../TransferPanel/NetworkListbox'
+
+export default function WalletConnectChainDropdown() {
+  const options = getSupportedNetworks(undefined, true)
+    .map(chainId => {
+      const wagmiChain = getWagmiChain(chainId)
+      const targetChainKey =
```

Comment:
Would it be possible to have an object:
```typescript
const chainIdToWalletConnectKey: { [key in ChainId]: string } = {
   ChainId.Ethereum: 'mainnet',
   . . .
}
```

and then we could just do:
```typescript
const targetChainKey = chainIdToWalletConnectKey[chainId]
```

Reply:
done